src/renderers/WebGLRenderer.d.ts

```diff
@@ -315,7 +315,7 @@ export class WebGLRenderer implements Renderer {
 
 	renderBufferDirect(
 		camera: Camera,
-		fog: Fog,
+		fog: IFog,
```

Comment:
I am uncertain about this, as I don't really know TypeScript. It didn't look correct before anyway, since FogExp2 was not included.